content/en/news/releases/1.13.x/announcing-1.13/_index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Announcing Istio 1.13
+linktitle: 1.13
+subtitle: Major Update
+description: Istio 1.13 release announcement.
+publishdate: 2022-02-08
```

Comment:
You'll want to update the date here as well. Assuming it may be today. Feb 9.

Reply:
This has been fixed.